server/sqlstore/stats.go

```diff
@@ -124,7 +123,7 @@ func (s *StatsStore) RunsFinishedBetweenDays(filters *StatsFilters, startDay, en
 	query := s.store.builder.
 		Select("COUNT(i.Id) as Count").
 		From("IR_Incident as i").
-		Where(sq.And{sq.Expr("i.EndAt > ?", startInMS), sq.Expr("i.EndAt < ?", endInMS)})
+		Where(sq.And{sq.Expr("i.EndAt > ?", startInMS), sq.Expr("i.EndAt <= ?", endInMS)})
```

Comment:
nit: I find it helpful to write this as:
```suggestion
		Where(sq.And{
		    sq.Expr("i.EndAt > ?", startInMS), 
		    sq.Expr("i.EndAt <= ?", endInMS),
		})
```